tools/scripts/ManifestBuilder.rb

```diff
@@ -356,7 +359,15 @@ def build_animation_metadata(animation_objects, previous_metadata)
 
       # Record target version in the metadata, so environments (and projects)
       # consistently reference the version they originally imported.
-      metadata['version'] = objects['png'].object.version_id
+      begin
+        metadata['version'] = objects['png'].object.version_id
+      rescue Aws::Errors::ServiceError => service_error
+        next <<-WARN
```

Comment:
What does `next <string>` do?

Reply:
I'm pretty sure it just functions like "return". `return "string"` I'm not sure why the original author used `next`. I will look into it.

Reply:
I looked into it. The reason we are using `next` instead of `return` is because this code used to be wrapped in a `.each` loop. When it was changed to `Parallel.map` all the `next` usage was not updated to use `return`. There isn't a functional difference.